easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.8.0-foss-2018a-Python-3.6.4.eb

```diff
@@ -13,12 +13,14 @@ patches = [
     'TensorFlow-1.5.0_swig-env.patch',
     'TensorFlow-%(version)s_remove-msse-hardcoding.patch',
     'TensorFlow-1.6.0_lrt-flag.patch',
+    'TensorFlow-1.8.0_toco_lrt-flag.patch',
```

Comment:
@migueldiascosta Wouldn't it make sense to collapse the two patches that add `-lrt`?